### DISCUSSION TITLE
Verdure for marketplaces

### DISCUSSION THREAD

**Discussion by matttbc** [vendurehq/vendure#508] [Q&A] [answered]
Hi Verdure community. We are a team of developers looking to help our local communities at this difficult time by building an online high street for small retailers to be able compete with big digital. Our challenge is that we will have multiple retailers in one experience. Has anyone extended Verdure to work as a marketplace?
Thanks

**Comment by michaelbromley** [comment, marked as answer]
Hi there! I would refer you to this thread: https://github.com/vendure-ecommerce/vendure/discussions/495 which discusses using Vendure for marketplaces.
